### PR TITLE
feat : added filter by price

### PR DESCRIPTION
**Pull request by aqsaaqeel**
Features implemented :

- Filter by price
  - HIgh to Low
  - Low to high

Next PR : Filter by rating and categories

You canview the features by clicking on any card on landing page and going to product listing page.
[thriftersstop.netlify.app](https://thriftersstop.netlify.app/)

### CODE REVIEW

File: src/reducerFuncs/priceReducer.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export function priceReducer(state, action){
+    switch(action.type){
+        case "SORT_PRICE":
+            return {...state, sortBy: action.payload}
+    }
+}
+
+export function itemList(sortBy, data){
+    switch (sortBy){
+        case "HIGH_TO_LOW" :
```
Comment (rehanmohiuddin) [inline src/reducerFuncs/priceReducer.jsx:12]: Make Constant I will be scalable and clean

File: src/index.js [hunk 2]
```diff
@@ -3,18 +3,21 @@ import ReactDOM from "react-dom";
...
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
-      <CartProvider>
-        <App />
-      </CartProvider>
+      <FilterProvider>
```
Comment (rehanmohiuddin) [inline src/index.js:16]: suggest: can make one index file for context Provider pass children Between all providers All App.js looks clean

File: src/Components/FilterNavbar/FilterNavbar.jsx [hunk 3]
```diff
@@ -78,7 +96,7 @@ export function FilterNavbar() {
                     </div>
                 </li>
                 <div className="small-space"></div>
-                <li>
+                {/* <li>
```
Comment (rehanmohiuddin) [inline src/Components/FilterNavbar/FilterNavbar.jsx:99]: remove comments

**Review by rehanmohiuddin — APPROVED** [review]

File: src/Components/FilterNavbar/FilterNavbar.jsx [hunk 4]
```diff
@@ -19,6 +25,18 @@ export function FilterNavbar() {
                         <a href="#">
                             <span className="title">Price</span>
                         </a>
+                        <div className="nav-sublists">
+                            <ul>
+                                <li>
+                                    <input type="radio" name="c1" id="c1" checked = {initialState.sortBy && initialState.sortBy === "LOW_TO_HIGH"} onClick={() => filterAction({ type : "SORT_PRICE", payload : "LOW_TO_HIGH"})}/>
```
Comment (Vivek-Py) [inline src/Components/FilterNavbar/FilterNavbar.jsx:31]: Really liking the structure, one thing you can do is make dispatch strings constants. Not necessary, just makes it easy to scale. Keep up!

**Review by Vivek-Py — APPROVED** [review]

**Pull request merged**